eviatar/enable_windows_features.ps1

```diff
@@ -0,0 +1,33 @@
+﻿function Install-WindowsFeatures {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string[]]$Features
+    )
+    
+    foreach ($Feature in $Features) {
+        Write-Host "Installing feature: $Feature"
+        Install-WindowsFeature -Name $Feature -IncludeAllSubFeature -IncludeManagementTools -Verbose
+    }
+}
+
+function Rename-Server {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$NewName
+    )
+
+    Write-Host "Renaming server to: $NewName"
+    Rename-Computer -NewName $NewName -Force
+}
+
+$features = Read-Host "Enter the list of Windows Features to install (comma-separated)"

```

Comment:
I've already mentioned the need for an operator... potentially risky!

Reply:
As mentioned before, depending on manual input can introduce risks. Automating input through parameters, environment variables, or configuration files would mitigate these risks.